aiida_vasp/parsers/file_parsers/poscar.py

```diff
@@ -48,11 +48,15 @@ def __init__(self, *args, **kwargs):
         super(PoscarParser, self).__init__(*args, **kwargs)
         self.precision = 12
         self._structure = None
+        self.poscar_options = None
         self.init_with_kwargs(**kwargs)
 
     def _init_with_precision(self, precision):
         self.precision = precision
 
+    def _init_with_poscar_options(self, poscar_options):
```

Comment:
Absolutely! I was thinking that this could be important for other things, as you might have seen I also opened an [issue](https://github.com/aiida-vasp/parsevasp/issues/51) in `parsevasp` about the need of sometimes have the coordinates written in Cartesian coordinates. In my fork I have made some modifications to make this possible (also in my aiida-vasp fork) so I think that those options could be use to let the user control this in a more granular manner.

Reply:
Great, thanks.